docs/developer/checkout/lines.mdx

```diff
@@ -1,10 +1,10 @@
 ---
-title: Managing products
+title: Managing lines items
 ---
 
 ### Adding a line to checkout
 
-To add an item to the cart, use [`checkoutLinesAdd`](api-reference/checkout/mutations/checkout-lines-add.mdx). Total prices will be updated automatically:
+To add an item to the cart, use [`checkoutLinesAdd`](api-reference/checkout/mutations/checkout-lines-add.mdx) to add lines to checkout. Total price will be updated automatically:
```

Comment:
> To add an item to the cart, use checkoutLinesAdd to add lines to checkout.

**polish**:
This sentence sounds quite repetitive. Is the addition of `...to add lines to checkout` really needed?